x/faucet/client/rest/rest.go

```diff
@@ -0,0 +1,12 @@
+package rest
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+)
+
+func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	registerQueryRoutes(cliCtx, r)
```

Comment:
We should have a `chainID` guard in place here. ie if chainID != sifchain it should register

Reply:
Added